packages/themes/types/appTheme.ts

```diff
@@ -1,6 +1,12 @@
+import { AppChromeTheme } from "../../appChrome/types";
+
 type Color = React.CSSProperties["color"];
+export type LocalTheme = AppChromeTheme & {
```

Comment:
single `&`? must be a TypeScript thing I'm unfamiliar with

Reply:
I was unfamiliar as well, it's an 'intersection type' so that it includes the types from AppChromeTheme and whatever is joined with the &.